packages/webui/src/client/ui/Status/SystemStatus.tsx

```diff
@@ -72,433 +73,434 @@ export function statusCodeToString(t: i18next.TFunction, statusCode: StatusCode)
 	}
 }
 
-export const DeviceItem = reacti18next.withTranslation()(
-	class DeviceItem extends React.Component<Translated<IDeviceItemProps>, IDeviceItemState> {
-		deviceTypeString() {
-			const t = this.props.t
-
-			switch (this.props.device.type) {
-				case PeripheralDeviceType.MOS:
-					return t('MOS Gateway')
-				case PeripheralDeviceType.PLAYOUT:
-					return t('Play-out Gateway')
-				case PeripheralDeviceType.MEDIA_MANAGER:
-					return t('Media Manager')
-				case PeripheralDeviceType.LIVE_STATUS:
-					return t('Live Status Gateway')
-				default:
-					return t('Unknown Device')
-			}
-		}
-		deviceVersions() {
-			const versions = this.props.device.versions
-			if (versions) {
-				return _.map(versions, (version, packageName) => {
-					return packageName + ': ' + version
-				}).join('\n')
-			}
+export function DeviceItem(props: IDeviceItemProps): JSX.Element {
+	const { t, i18n, ready } = reacti18next.useTranslation()
```

Comment:
```suggestion
	const { t, i18n, ready } = useTranslation()
```

Reply:
In this file there was already a `reacti18next.withTranslation` ;)